src/Stripe/Entities/StripeTransfer.cs

```diff
@@ -58,6 +58,12 @@ internal object InternalBalanceTransaction
         [JsonProperty("description")]
         public string Description { get; set; }
 
+        [JsonProperty("destination")]
+        public string Destination { get; set; }
```

Comment:
This is an expandable property - so the service needs to account for that (with a bool).